esp/src/package.json

```diff
@@ -38,7 +38,9 @@
     "@hpcc-js/html": "2.8.2",
     "@hpcc-js/map": "2.14.7",
     "@hpcc-js/other": "2.13.12",
-    "@hpcc-js/tree": "2.9.1",
+    "@hpcc-js/tree": "2.11.0",
+    "@hpcc-js/codemirror": "2.12.3",
+    "@hpcc-js/phosphor": "2.13.15",
     "@material-ui/core": "4.8.3",
     "clipboard": "2.0.4",
     "codemirror": "5.50.2",
```

Comment:
@jbrundage When editing package.json - you also need to run "npm install" local AND then commit package-lock.json at the same time (which is why your getting smoketest failures.